### DISCUSSION TITLE
How can I disable/enable a custom Helper in CLI?

### DISCUSSION THREAD

**Discussion by alaz-aura** [codeceptjs/CodeceptJS#4025] [Q&A] [answered]
Howdy 👋 

So I have a custom API helper created. I want to enable/disbale the active state of the helper when running with CLI.

Here's an example from my `codecept.conf.ts` helper 
```
"CustomHelper": {
      require: './helpers/custom_helper.ts', // path to module
      'enabled': true
    }
```

Although it's `true` now, I want to toggle the `enabled` flag. Any ideas how I can do this?

**Comment by jancorvus** [comment]
https://codecept.io/commands/#run
```
npx codeceptjs run --override '{ "helpers": {"CustomHelper": {"enabled": "false"}}}'
```
If it does not work right away, try to escape some `"` as `\"`

**Reply by alaz-aura** [reply]
ah thanks for the attempt, doesn't look like it worked for me unfortunately.

**Comment by kobenguyent** [comment, marked as answer]
Hi I guess CodeceptJS currently only supports enabling/disabling the plugins.

**Reply by alaz-aura** [reply]
yes seems that way. Perhaps I need to convert my helper logic to be a plugin instead.
